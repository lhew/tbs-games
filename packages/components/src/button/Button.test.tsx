import React from 'react';
import TestRenderer from 'react-test-renderer'; // ES6
import Button from '.';

test('Button scenarios', () => {
  const defaultBtn = TestRenderer.create(<Button>Hello</Button>);
  expect(defaultBtn).toMatchSnapshot();
});
