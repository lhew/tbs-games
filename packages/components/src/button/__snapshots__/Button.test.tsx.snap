// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button scenarios 1`] = `
<span
  className="sc-AxjAm IUULe"
>
  Hello
</span>
`;
